lib/objects/objectsUtils.js

```diff
@@ -167,7 +168,7 @@ function insert(objects, id, attName, _ignore, options, _obj, callback) {
             if (memStore[id + '/' + attName] !== undefined) {
                 delete memStore[id + '/' + attName];
             }
-            callback && setImmediate(callback, error, null);
+            return tools.maybeCallbackWithError(callback, error, null);
```

Comment:
we loose the setImmediate here or will it be implicitly?

Reply:
(and other places too)

Reply:
https://github.com/ioBroker/ioBroker.js-controller/blob/master/lib/tools.js#L2273